managers/factories.py

```diff
@@ -0,0 +1,13 @@
+import factory
+import factory.fuzzy
+
+from managers.models import Project
+
+
+class ProjectFactory(factory.DjangoModelFactory):
```

Comment:
Please add simple test for this factory

Reply:
As discussed, we don't want to test factories.